docs/podman-pod-stop.1.md

```diff
@@ -0,0 +1,53 @@
+% podman-pod-stop "1"
+
+## NAME
+podman\-pod\-stop - Stop one or more containers
+
+## SYNOPSIS
+**podman pod stop** [*options*] *pod* ...
+
+## DESCRIPTION
+Stop containers in one or more pods.  You may use pod IDs or names as input.
+
+## OPTIONS
+
+**--all, -a**
+
+Stops all pods
+
+**--cleanup**
```

Comment:
Why?

Reply:
I'm not sure!

Reply:
Kill it.

Reply:
for clarity: kill it is in kill this option, or kill it as in that is the reason why one would use --cleanup?